diff --git a/plotjuggler_base/src/plotzoomer.cpp b/plotjuggler_base/src/plotzoomer.cpp
index 54b1b49..71dbe4e 100644
--- a/plotjuggler_base/src/plotzoomer.cpp
+++ b/plotjuggler_base/src/plotzoomer.cpp
@@ -63,7 +63,7 @@ void PlotZoomer::widgetMouseMoveEvent(QMouseEvent* me)
           _zoom_enabled = true;
           this->setRubberBand(RectRubberBand);
           this->setTrackerMode(AlwaysOff);
-          QPen pen(parentWidget()->palette().foreground().color(), 1, Qt::DashLine);
+          QPen pen(parentWidget()->palette().windowText().color(), 1, Qt::DashLine);
           this->setRubberBandPen(pen);
           QApplication::setOverrideCursor(zoom_cursor);
         }
