diff --git a/3rdparty/qwt/src/qwt_picker.h b/3rdparty/qwt/src/qwt_picker.h
index 3af1012..81e9a1f 100644
--- a/3rdparty/qwt/src/qwt_picker.h
+++ b/3rdparty/qwt/src/qwt_picker.h
@@ -18,19 +18,19 @@
 class QwtPickerMachine;
 class QwtWidgetOverlay;
 class QwtText;
-class QWidget;
-class QMouseEvent;
-class QWheelEvent;
-class QKeyEvent;
-class QPainter;
-class QPen;
-class QFont;
-class QRegion;
-class QPainterPath;
-class QPoint;
-class QRect;
-class QSize;
-class QPolygon;
+#include <QWidget>
+#include <QMouseEvent>
+#include <QWheelEvent>
+#include <QKeyEvent>
+#include <QPainter>
+#include <QPen>
+#include <QFont>
+#include <QRegion>
+#include <QPainterPath>
+#include <QPoint>
+#include <QRect>
+#include <QSize>
+#include <QPolygon>
 
 /*!
    \brief QwtPicker provides selections on a widget
